models/aether_lat-lon/work/input.nml

```diff
@@ -0,0 +1,325 @@
+&probit_transform_nml
+   /
+
+&algorithm_info_nml
+   qceff_table_filename = ''
+   /
+
+&quality_control_nml
+/
+
+&state_vector_io_nml
+   /
+
+&perfect_model_obs_nml
+   read_input_state_from_file = .true.
+   single_file_in             = .false.
+   input_state_files          = 'wrfinput_d01'
```

Comment:
in pmo: the input_state_files is set to 'wrfinput_d01'. We probably need to change that

Reply:
That will need to be changed, but pmo is not in the scope of this stage of development,
so we haven't looked at that.  I'll replace wrfinput_d01 with "pmo not ready for use".